plugins/generator-1.15.2/forge-1.15.2/procedures/block_move_by_piston.java.ftl

```diff
@@ -0,0 +1 @@
+(PistonBlock.canPush(new BlockState(((world.getBlockState(new BlockPos((int)${input$x},(int)${input$y},(int)${input$z})))).getBlock(), ImmutableMap.of()), (World) world, new BlockPos((int)${input$x},(int)${input$y},(int)${input$z}), ${input$direction}, false, ${input$direction}))
```

Comment:
You don't have to create a new BlockState, getting it from world is enough (same for 1.16)